scripts/generic-ci-tests.sh

```diff
@@ -80,8 +80,8 @@ case "$TEST_SUITE" in
         paver run_pylint -l $PYLINT_THRESHOLD > pylint.log || { cat pylint.log; EXIT=1; }
 
         mkdir -p reports
-        echo "Finding jshint violations and storing report..."
-        paver run_jshint -l $JSHINT_THRESHOLD > jshint.log || { cat jshint.log; EXIT=1; }
+        echo "Finding ESLint violations and storing report..."
+        paver run_eslint -l $ESLINT_THRESHOLD > eslint.log || { cat eslint.log; EXIT=1; }
```

Comment:
Can we just make the jobs report both, and for now, the eshint one won't exist, and that will be ok? Then we won't break everyone w/ an outstanding PR when this lands.